progress-bar.dust

```diff
@@ -0,0 +1,19 @@
+<div {?id}id="{.}"{/id} class="c-progress-bar c--{state} {?isRadial}c--radial{/isRadial} {class}" role="progressbar" aria-valuenow="{value}" aria-valuemin="0" aria-valuemax="1" aria-valuetext="{label}" data-adaptivejs-component="stencil-progress-bar">
```

Comment:
Okay, that makes sense to me. I believe the attributes were in this order when I generated the component, so we'll need to make sure to change this in the generator.